apps/ui/src/components/SpaceDelegates.vue

```diff
@@ -134,7 +134,7 @@ watchEffect(() => setTitle(`Delegates - ${props.space.name}`));
           v-if="loaded && (delegates.length === 0 || failed)"
           class="px-4 py-3 flex items-center space-x-2"
         >
-          <IH-exclamation-circle class="inline-block" />
+          <IH-exclamation-circle class="inline-block mr-1" />
```

Comment:
I would just put the `<template>` below inside a `<span>`, which will auto-enable the spacing set by `space-x-2` in the parent element

Reply:
Should also add a `shrink-0` to the icon